packages/reporting-server/rest_server/default_config.py

```diff
@@ -3,7 +3,7 @@
     "host": "127.0.0.1",  # ip or hostname to bind the socket to
     "port": 8002,
     "port_fluentd": 8003,
-    "db_url": "sqlite://:memory:",
+    "db_url": "postgres://postgres:postgres@localhost:5432/reporting",
```

Comment:
Does this mean that it is no longer possible to `npm start` the server without a running postgres instance?

Reply:
Yes or any other persistent database